helm-chart/templates/cronjobs/sitemaps.yaml

```diff
@@ -0,0 +1,63 @@
+{{if eq (substr 0 4 .Values.deployEnv) "prod"}}
```

Comment:
I think this can come out as its not a core part of the app

Reply:
Should I be removing the condition or the whole cronjob?